src/platform/common/utils/localize.ts

```diff
@@ -1262,6 +1436,8 @@ function getString(key: string, defValue?: string) {
         collection = loadedCollection;
     }
     let result = collection[key];
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    result = (result as any)?.message || result;
```

Comment:
Since `package.nls.json` now can return objects, we should retrieve the message from those objects in such cases.